datajunction-clients/python/Makefile

```diff
@@ -7,14 +7,7 @@ lint:
 test:
 	pdm run pytest --cov=datajunction --cov-report=html -vv tests/ --doctest-modules datajunction --without-integration --without-slow-integration ${PYTEST_ARGS}
 
-version:
-	@poetry version $(v)
-	@git add pyproject.toml
-	@git commit -m "v$$(poetry version -s)"
-	@git tag v$$(poetry version -s)
-	@git push
-	@git push --tags
-	@poetry version
-
-release:
-	@poetry publish --build
+dev-release:
```

Comment:
Should this also include an alpha-release option, since we've been mostly doing alpha releases?

Reply:
I don't think so, because you only want to do the Alpha (which is our current cycle) in a synchronized fashion and not individually.